packages/tests/src/test/basic.ts

```diff
@@ -0,0 +1,14 @@
+import { assertElementLooks } from '../helpers/assertions'
+
+describe('Bigcommerce page', () => {
+    it('should have the right title', async () => {
+        // tslint:disable-next-line:no-console
```

Comment:
It looks like we are going to be using console.log a lot in these tests, I'd suggest configuring the linter to disable this specific rule instead of disabling every line where its used

Reply:
✅